### ISSUE TITLE
Usage examples on help page

### ISSUE THREAD

**Issue by Gig77** [rabix/bunny#256]
Couple of bunny usage examples in the 'rabix --help' output would be helpful to get the syntax quicker. Took me a bit to figure out the exact command line syntax for providing CWL inputs after the empty '--' parameter.

**Comment by StarvingMarvin** [comment]
True. We are also working on documentation, so these things should be resolved soon
